libbeat/scripts/Makefile

```diff
@@ -335,9 +335,11 @@ ifneq ($(shell [[ $(BEAT_NAME) == libbeat || $(BEAT_NAME) == metricbeat ]] && ec
 	go run  ${ES_BEATS}/dev-tools/cmd/asset/asset.go -pkg include -in fields.yml -out include/fields.go $(BEAT_NAME) 
 endif
 
+ifneq ($(shell [[ $(BEAT_NAME) == libbeat ]] && echo true ),true)
```

Comment:
@kvch Should we merge it first without this fix to see what error we get?

Reply:
yep. dropped the patch.